TESTS/netsocket/dns/main.cpp

```diff
@@ -204,7 +204,9 @@ Case cases[] = {
     Case("ASYNCHRONOUS_DNS", ASYNCHRONOUS_DNS),
     Case("ASYNCHRONOUS_DNS_SIMULTANEOUS", ASYNCHRONOUS_DNS_SIMULTANEOUS),
     Case("ASYNCHRONOUS_DNS_SIMULTANEOUS_CACHE", ASYNCHRONOUS_DNS_SIMULTANEOUS_CACHE),
+#if NSAPI_PPP_AVAILABLE
```

Comment:
I don't think you planned to ignore test for ethernet and wifi connections. Was there some particular reason why this is not a valid case for AT cellular connections?

Reply:
Changed the flag to MBED_CONF_CELLULAR_OFFLOAD_DNS_QUERIES.